### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by noah-curran**
### Code Review Bot Information
_Below you will find information created by this bot!_

**Reviewers:**

**Parent Issue:** #49

**Children Issues:**

### CODE REVIEW

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

File: about.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ title: About
 permalink: /about/
 ---
 
-I study Computer Science and Math at Purdue University, intending to graduate in May of 2020. Afterwards, I aim to attend graduate school to obtain a PhD in Computer Science.
\ No newline at end of file
+I study Computer Science and Math at Purdue University, intending to graduate in May of 2020. Afterwards, I aim to attend graduate school to obtain a PhD in Computer Science.
+
+Test line here.
```
Comment (noah-curran) [inline about.md:9]: @cam2-code-review-bot this line was a test.

**Comment by noah-curran** [comment]
@cam2-code-review-bot test

**Comment by noah-curran** [comment]
@cam2-code-review-bot test

File: about.md [hunk 2]
```diff
@@ -4,4 +4,6 @@ title: About
 permalink: /about/
 ---
 
-I study Computer Science and Math at Purdue University, intending to graduate in May of 2020. Afterwards, I aim to attend graduate school to obtain a PhD in Computer Science.
\ No newline at end of file
+I study Computer Science and Math at Purdue University, intending to graduate in May of 2020. Afterwards, I aim to attend graduate school to obtain a PhD in Computer Science.
```
Comment (noah-curran) [inline about.md:7]: @cam2-code-review-bot this is a test comment

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot hello

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "this is a title" "this is an issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "this is a title" "this is an issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "this is a title" "this is an issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "this is a title" "this is an issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "this is a title" "this is an issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "test title openissue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "test title openissue2" "test desc"

**Comment by noah-curran** [comment]
@cam2-code-review-bot init 4

**Comment by noah-curran** [comment]
@cam2-code-review-bot 28

**Comment by noah-curran** [comment]
@cam2-code-review-bot init 28

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue" "description of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot init

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "title of issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot openissue "another title" "some body goes here **yay**"

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue "title"

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 hello

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 hello

**Comment by cam2-code-review-bot-2** [comment]
Hello human!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue "title"

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue "title here" "now here is some **body text**"

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by cam2-code-review-bot-2** [comment]
Command failed. Please use format `@bot-name "<title of new issue>" ["body of new issue"]`.

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue "test issue"

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue

**Comment by cam2-code-review-bot-2** [comment]
Command failed. Please use format `@bot-name openissue "<title of new issue>" ["<body of new issue>"]`.

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue "title"

**Comment by cam2-code-review-bot-2** [comment]
Command failed. Please first initialize the repository by using `@bot-name init` and then try again.

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init

**Comment by cam2-code-review-bot-2** [comment]
Command failed. This issue has already been initialized.

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 48

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 48

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 48

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 48

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 openissue "title"

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 100

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 100

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 100

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 100

**Comment by cam2-code-review-bot-2** [comment]
Command failed. The provided parent issue does not exist.

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by cam2-code-review-bot-2** [comment]
Command failed. The provided parent issue does not exist.

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by noah-curran** [comment]
@cam2-code-review-bot-2 init 49

**Comment by cam2-code-review-bot-2** [comment]
🤖 The bot has initialized this issue!

**Pull request closed without merging**